frontend/src/metabase-lib/temporal_bucket.ts

```diff
@@ -83,3 +83,7 @@ export function formatRelativeDateRange({
     "include-current": includeCurrent,
   });
 }
+
+export function availableTemporalUnits(): string[] {
```

Comment:
We can also update `Lib.describeTemporalUnit` signature 

```suggestion
export function availableTemporalUnits(): BucketName[] {
```